server/src/main/java/org/elasticsearch/rest/RestController.java

```diff
@@ -340,6 +349,11 @@ private void tryAllHandlers(final RestRequest request, final RestChannel channel
         handleBadRequest(uri, requestMethod, channel);
     }
 
+    public static boolean isV7Compatible(ToXContent.Params params) {
```

Comment:
Why is this V7 specific? Aren't the compatible params version agnostic?

Reply:
good point, fixed